### DISCUSSION TITLE
What do you often buy at Daily Yamazaki?

### DISCUSSION THREAD

**Discussion by Haru39M** [NAIST-SE/AttractingContributors#1155] [2024 Attraction Contributions]
https://github.com/Haru39M/AttractingContributors

**Comment by Ryota-keda** [comment]
I just made PR.
Could you contribute to my project?
https://github.com/Ryota-keda/Baseball

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by r-nishioka** [comment]
I just made PR.
Could you contribute to my project?
[Where would you like to trip in Japan?](https://github.com/r-nishioka/AttractingContributors-Trip.git)

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1148

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
